client/css/style.css

```diff
@@ -573,6 +573,7 @@ button {
 }
 #chat .wrap,
 #chat .text a {
+	cursor: pointer;
```

Comment:
Ah, very good point! I had forgotten about this oddity. Well, the HTML5 spec does say that [the `a` element, absent an `href`, is not an hyperlink but a placeholder](http://www.w3.org/TR/html5/text-level-semantics.html#the-a-element).

[This Stack Overflow](http://stackoverflow.com/a/11145021/1935861) is very well built and even links to WAI-ARIA!

My conclusion on this (also after reading [this](http://css-tricks.com/when-and-when-not-to-use-an-anchor-tag/)) is that we should not go for an `a` link. We should either go for a `button` or a `span` with the right ARIA role and focus behavior. At the moment links on the left menu are using a `span`, links on the right menu are using a `button`. Please go for a `button` and if you realize it's hell to style (not sure how it behaves with long statements that go to a second line) then switch back to a `span`:

```html
<button role="button" tabindex="0">...</button>
```

This is definitely the right way to go for in-app "links" and I'll make sure we go for the right route in subsequent choices. And we keep `a` links for actual hyperlinks.

Does that make sense to you? Let me know if you have any questions, and sorry for being slow at first :-)